File: Year3/Programming_Language_6237/Ex2/FToC.go
package main

import "fmt"

func main() {
	var fahrenheit float64
	fmt.Print("Enter degree in Fahrenheit: ")
	fmt.Scan(&fahrenheit)
	fmt.Printf("%.1f Fahrenheit = %.1f Celsius\n", fahrenheit, (5.0/9.0)*(fahrenheit-32))
}

File: Year3/Programming_Language_6237/Ex2/FToC.cpp
#include <iostream>

using namespace std;

int main() {
      double fahrenheit,celsius;
      cout << "enter degree in fahrenheit: ";
      cin >> fahrenheit;
      celsius =  (5.0/9.0) * (fahrenheit - 32);
      cout << fahrenheit  << " fahrenheit =  " << celsius << " celsius" ;
}